extension/chrome/elements/compose.ts

```diff
@@ -154,7 +154,7 @@ Catch.try(async () => {
             if (!parsed.keys[0]) {
               Catch.log('Dropping found but incompatible public key', { for: email, err: parsed.err ? ' * ' + parsed.err.join('\n * ') : undefined });
               lookupResult.pubkey = null; // tslint:disable-line:no-null-keyword
-            } else if (! await parsed.keys[0].getEncryptionKey()) {
+            } else if (! await parsed.keys[0].getEncryptionKey() && !Pgp.key.usableButExpired(parsed.keys[0])) { // Not to skip expired keys
```

Comment:
Should be `! await Pgp.keyusableButExpired`

Please also add a test. Thanks!

Reply:
Actually try this:
```js
} else if (! await Pgp.key.usable(...) && ! await Pgp.key.expired(...)) {
```
It should be more accurate.

Reply:
Actually - please not just test that it becomes orange, but try to send a message out. There is supposed to be a modal dialog asking the user to confirm to send the message even though key is expired - please test that it shows the dialog too, and then that the message gets sent properly.

Reply:
Sorry, but we can't send emails if there is at least on expired email.
Do I have to allow it?

Reply:
I wrote the tests, but I've got an issue:
When sending the message with an expired pub. Key it shows the modal that says there is expired key then I click `Ok` and it tries to send the message but openpgp.js returns me this:
![image](https://user-images.githubusercontent.com/49104848/64390737-ba1d0000-d04e-11e9-9c8d-82008026c0bf.png)

Do you know what this means?

Also, I found two ways I can get an error when sending an email with an expired recipient:
1) It's the way where I have `confirm` modal dialog
2) This one when I see error modal immediately:
```TS
    if (usableTimeFrom > usableTimeUntil) { // used public keys have no intersection of usable dates
      await Ui.modal.error('The public key of one of your recipients has been expired for too long.\n\nPlease ask the recipient to send you an updated Public Key.');
      throw new ComposerResetBtnTrigger();
    }
```

I guess I need to test this way too. But I don't know how to get the key that will fit this expression.

Reply:
I think I'll take a look at this, it's getting somewhat complex.

Reply:
Ok, I pushed the changes